### DISCUSSION TITLE
How to setup Directory in url tag

### DISCUSSION THREAD

**Discussion by rmk177** [pelicanmapping/osgearth#1894] [Support Q&A] [answered]
Good day! long time ago I simply setup directory, like this
<elevation name="srtm_elevation" driver="gdal">
        <url>./maps/Height/srtm</url>
</elevation> 
and all works fine. All data from this directory loaded fine.
But in osgearth master I can't do this. 
Could you please give me some advice how can I load files from directory
Thanc in advance
Bye

**Comment by gwaldron** [comment, marked as answer]
We no longer support loading an entire directory like that. Instead, you need to use the `gdalbuildvrt` tool to make a GDAL VRT file.  https://gdal.org/programs/gdalbuildvrt.html
